backend/preprocessing/hysteresis_loops.py

```diff
@@ -115,7 +113,7 @@ def PolyArea(x, y):
     Stiffness.insert(0, np.nan)
 
     # CREEP
-    ### Creep computation
+    # Creep computation
```

Comment:
Creep

Reply:
The comments below seem a bit weird. Is it documentation ? Is it dead code ? Should it help me understand better how strain should be computed ?
Where does the math come from ? Maybe I missed out on some documentation, but I would have like to have a link at least to a reference implementation or paper explaining the math. Please enlighten us! 

OOw, I got my answer in the header: 'Fatigue_test_dashboard/Hysteresis_loops.py (written by Scott)'

Soo.. What's the Fatigue_test_dashboard ? and Who's Scott ?

Reply:
I really don't know. It's _magic code_ received from the lab.